pkg/kubelet/volume/host_path/host_path.go

```diff
@@ -54,6 +54,12 @@ func (plugin *hostPathPlugin) CanSupport(spec *api.Volume) bool {
 	return false
 }
 
+func (plugin *hostPathPlugin) GetAccessModes() []api.AccessModeType {
+	return []api.AccessModeType{
+		api.ReadWriteOnce,
```

Comment:
It's not clear what this means for local-only volumes like hostPath and emptyDir.  Why does gitRepo return no modes, but empty returns RW-Once? And why would hostDir not be RW-Many?